src/schema/rules/tabular_data/task.yaml

```diff
@@ -10,6 +10,11 @@ TaskEvents:
     response_time: optional
     HED: optional
     stim_file: optional
+    channel:
+      level: optional
+      description_addendum: |
```

Comment:
I don't completely understand why this is in the task.yaml file.  Is task.yaml where it determines if tabular data applies to a particular modality?

Reply:
`task.yaml` is an arbitrary name. It could have been called `events.yaml`, but the rule `schema.rules.tabular_data.task.TaskEvents` defines what are valid columns in an `_events.tsv` file. The filename construction (and hence what modalities apply) are defined in the filename rules:

https://github.com/bids-standard/bids-specification/blob/426de564d995f2316ccf13ca66999623186f5bac/src/schema/rules/files/raw/task.yaml#L2-L19

https://github.com/bids-standard/bids-specification/blob/426de564d995f2316ccf13ca66999623186f5bac/src/schema/rules/files/raw/task.yaml#L2-L19